static/bidder-info/vimayx.yaml

```diff
@@ -0,0 +1,5 @@
+endpoint: "http://vimayx-prebid.smart-hub.io/pbserver/?seat={{.AccountID}}&token={{.SourceId}}"
+aliasOf: "smarthub"
+userSync:
```

Comment:
If the host has configured user syncing for smarthub, it would be inherited for this alias. The only reason to define a userSync section here is if VimayX intends to use a separate user sync.

@SmartHubSolutions Please advise on the intent.

Reply:
You're right, I removed userSync it's not necessary